src/mono/wasm/runtime/jiterpreter-feature-detect.ts

```diff
@@ -2,43 +2,60 @@ import {
     WasmValtype, WasmOpcode, WasmSimdOpcode
 } from "./jiterpreter-opcodes";
 import {
-    WasmBuilder,
+    WasmBuilder, getRawCwrap
 } from "./jiterpreter-support";
 
-export function compileDoJitCall () : WebAssembly.Module {
-    const builder = new WasmBuilder(0);
-    builder.defineType("jit_call_cb", {
+export let doJitCallBuilder : WasmBuilder | undefined = undefined;
+
+// FIXME: We no longer actually use this function, it only serves as a feature check at this point.
```

Comment:
We should store `canRunSIMD` and `canRunEH` into `runtimeHelpers` during `ensureUsedWasmFeatures`

And then you could drop this. I drafted it in https://github.com/dotnet/runtime/pull/92480 but I'm not sure I'm going to merge it. It would be cool to do it in this PR.

![image](https://github.com/dotnet/runtime/assets/271576/4e488116-da4f-4a3c-90eb-8451ac4eedcc)